### ISSUE TITLE
Instructor view session results (course-wide): Improve accessibility of grouped responses

### ISSUE THREAD

**Issue by zhaojj2209** [TEAMMATES/teammates#12285]
TEAMMATES provides multiple ways of grouping session results for instructors, an example being the following (giver --> recipient --> question):
<img width="1149" alt="Screenshot 2023-03-28 at 16 00 36" src="https://user-images.githubusercontent.com/54243224/228172258-20257b82-d916-496b-8edb-b7f7a4798fb7.png">

The grouped responses views are currently not accessible to users using assistive technologies. At minimum, the following should be addressed:
- Dropdowns should semantically be buttons so that they can be picked up by assistive technologies.
- Each level of the dropdown should be an appropriate heading (e.g. `h2` for section, `h3` for team, ...).
- The comment button should have a corresponding label (e.g. "Add Comment").

**Comment by athakaras** [comment]
Hello! I tried finding the respective file but I couldn't. Is it inside the src\web\app\pages-session path ?

**Comment by samuelfangjw** [comment]
@athakaras The code can be found at `src/web/app/pages-instructor/instructor-session-result-page`.

To access this feature, from the instructor home pages, click on results -> view results (course-wide).

<img width="357" alt="Screenshot 2023-04-25 at 5 41 31 PM" src="https://user-images.githubusercontent.com/60355570/234238290-875abb97-da4f-4968-80aa-47f0c1f18f8d.png">


Then change the view as follows:

<img width="620" alt="Screenshot 2023-04-25 at 5 39 43 PM" src="https://user-images.githubusercontent.com/60355570/234237826-79c254a0-18b4-430a-a675-7d77c2a304c2.png">

**Comment by athakaras** [comment]
@samuelfangjw hello!

I've been searching the code for some time and I found that in 
```src\web\app\pages-instructor\instructor-session-result-page\instructor-session-result-page.component.html ``` file, 
the questions come from the following properties;

tm-instructor-session-result-question-view
tm-instructor-session-result-grq-view
tm-instructor-session-result-rgq-view
tm-instructor-session-result-gqr-view
tm-instructor-session-result-rqg-view

do you know where i can find the above files? :)

**Comment by zhaojj2209** [comment]
You already have the selectors that identify each component, to find the corresponding files, just search the codebase to find the component with the same selector.

To make things easier, the components are in the same `pages-instructor\instructor-session-result-page` folder.

**Comment by athakaras** [comment]
@zhaojj2209 sorry i think i didn't express correctly what I wanted to say. I have found the components but i need the files of the questions that are imported (maybe it is a question components) where the tags , the comment button and the drop-down menu are...

**Comment by zhaojj2209** [comment]
You may look in the HTML files in order to find which components are being used, e.g. `instructor-session-result-rqg-view.component.html` uses the `tm-gqr-rqg-view-responses` component. You can then find the files using the method I've described above.

For convenience, most of the components should be in the `components/question-responses` folder.

**Comment by athakaras** [comment]
@zhaojj2209 just raised a PR. I hope it helps:)
